### DISCUSSION TITLE
Google -> Kagi

### DISCUSSION THREAD

**Discussion by inhzus** [mshibanami/redirect-web#47] [Redirect Rules]
We now cannot set [Kagi](https://kagi.com) as the default search engine for Safari.

Redirect from: `^https://www\.google\.com/search\?q=([^&]*).*$` (Mode: regular expression)
Redirect to: `https://kagi.com/search?q=$1`
Application: Default (Safari)
Capture Group Processing:
Excluded URL Patterns:
Examples: `https://www.google.com/search?q=hello&ie=UTF-8&oe=UTF-8&hl=en&client=safari` -> `https://kagi.com/search?q=hello`

**Comment by mshibanami** [comment]
This is great! Thanks @inhzus. If you don't mind, could I add it to the Library in the app as a useful pre-defined rule so other users can easily download it?

**Reply by inhzus** [reply]
Sure.

**Reply by mshibanami** [reply]
Added it.
I modified the original settings a bit so it works on each regions' Google Search. Thanks!
